docs/model-repos.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Model Repos docs
+---
+
+<h1>Model Repos docs</h1>
+
+## What are model cards and why are they useful?
+
+The model cards are markdown files that accompany the models and provide very useful information. They are extremely important for discoverability, reproducibility and sharing! They are the `README.md` file in any repo.
+
+## When sharing a model, what should I add to my model card?
+
+The model card should describe:
+- the model
+- its intended uses & potential limitations, including bias and ethical considerations as detailed in [Mitchell, 2018](https://arxiv.org/abs/1810.03993)
+- the training params and experimental info (you can embed or link to an experiment tracking platform for reference)
+- which datasets did you train on and your eval results
+
+If needed you can find a template [here](https://github.com/huggingface/model_card).
```

Comment:
You can also link to the specification: https://raw.githubusercontent.com/huggingface/huggingface_hub/main/modelcard.md